docs/_heads/include_resource.md

```diff
@@ -3,33 +3,8 @@ layout: default
 class: Header
```

Comment:
Isn't this file already changed by a merged PR?

Reply:
will be changed more complete soon